v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/templates/bigtablechangestreamstobigquery/model/ChangelogColumn.java

```diff
@@ -19,38 +19,49 @@
 
 /** Static metadata class for all changelog columns. */
 public enum ChangelogColumn {
-  ROW_KEY_STRING("row_key", StandardSQLTypeName.STRING.name(), true, false),
-  ROW_KEY_BYTES("row_key", StandardSQLTypeName.BYTES.name(), true, false),
-  MOD_TYPE("mod_type", StandardSQLTypeName.STRING.name(), true, false),
-  COMMIT_TIMESTAMP("commit_timestamp", StandardSQLTypeName.TIMESTAMP.name(), true, false),
-  COLUMN_FAMILY("column_family", StandardSQLTypeName.STRING.name(), true, false),
-  COLUMN("column", StandardSQLTypeName.STRING.name(), false, false),
-  TIMESTAMP("timestamp", StandardSQLTypeName.TIMESTAMP.name(), false, false),
-  TIMESTAMP_NUM("timestamp", StandardSQLTypeName.INT64.name(), false, false),
-  VALUE_STRING("value", StandardSQLTypeName.STRING.name(), false, false),
-  VALUE_BYTES("value", StandardSQLTypeName.BYTES.name(), false, false),
-  TIMESTAMP_FROM("timestamp_from", StandardSQLTypeName.TIMESTAMP.name(), false, false),
-  TIMESTAMP_FROM_NUM("timestamp_from", StandardSQLTypeName.INT64.name(), false, false),
-  TIMESTAMP_TO("timestamp_to", StandardSQLTypeName.TIMESTAMP.name(), false, false),
-  TIMESTAMP_TO_NUM("timestamp_to", StandardSQLTypeName.INT64.name(), false, false),
-  IS_GC("is_gc", "BOOL", true, true),
-  SOURCE_INSTANCE("source_instance", StandardSQLTypeName.STRING.name(), true, true),
-  SOURCE_CLUSTER("source_cluster", StandardSQLTypeName.STRING.name(), true, true),
-  SOURCE_TABLE("source_table", StandardSQLTypeName.STRING.name(), true, true),
-  TIEBREAKER("tiebreaker", StandardSQLTypeName.INT64.name(), true, true),
+  ROW_KEY_STRING("row_key", StandardSQLTypeName.STRING.name(), true, false, null),
```

Comment:
Very nit

Just a note that this is a bit hard to read without an IDE that has inlay parameters. Creating a dummy constant NO_DEFAULT_VALUE would make this less worse

Reply:
Let's incorporate this into other upcoming PRs, really want to merge this stuff

Reply:
I implemented this, but as you can see it's a bit awkward since I can't use static constants for enum member constructors.